.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  release:
+    if: ${{ github.event.label.name == 'release dev tag' }}
+    runs-on: ubuntu-22.04
+    env:
+      HUSKY: 0
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup PNPM
+        uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
+        with:
+          run_install: |
+            - args: [--frozen-lockfile]
+      - name: Check for changeset
+        run: pnpm exec changeset status --since origin/main
+      - name: Create .npmrc
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
```

Comment:
question: Can this action be run for external contributors? I can see that someone adds additional action that reads this file and steals our NPM_TOKEN 🙈

Reply:
I changed this setting - so we have to grant access for people outside of Saleor so then can run our workflows on their forks:

![CleanShot 2024-03-15 at 12 06 58@2x](https://github.com/saleor/macaw-ui/assets/9116238/ad7940f9-3185-49cb-a2ed-1437395f2a3d)